Weights & Biases tracking enabled.
FP16 Run: False
Dynamic Loss Scaling: True
Distributed Run: False
cuDNN Enabled: True
cuDNN Benchmark: False
/scratch/md5121/okhatisathi/tacotron2/stft.py:67: FutureWarning: Pass size=1024 as keyword args. From version 0.10 passing these as positional arguments will result in an error
  fft_window = pad_center(fft_window, filter_length)
/scratch/md5121/okhatisathi/tacotron2/layers.py:51: FutureWarning: Pass sr=22050, n_fft=1024, n_mels=80, fmin=0.0, fmax=8000.0 as keyword args. From version 0.10 passing these as positional arguments will result in an error
  sampling_rate, filter_length, n_mel_channels, mel_fmin, mel_fmax)
An error occurred during training: name 'use_wandb' is not defined
Traceback (most recent call last):
  File "<ipython-input-8-b83e5e1a4ed6>", line 36, in <module>
    hparams=hparams
  File "/scratch/md5121/okhatisathi/tacotron2/train.py", line 208, in train
    if use_wandb:
NameError: name 'use_wandb' is not defined